web/static/js/components/delete_button.js

```diff
@@ -3,7 +3,7 @@ import React from 'react'
 class DeleteButton extends React.Component {
   render () {
     return (
-    <div className="thesis-button delete">
+    <div onClick={this.props.onPress} className="thesis-button delete">
```

Comment:
This messed me up for a while. Couldn't figure out why my `onClick` wasn't being triggered, until I realized I wasn't setting it here.